app/utils/transaction_view.js

```diff
@@ -121,17 +141,21 @@ module.exports = {
 
   buildPaymentView: function (chargeData, eventsData) {
     eventsData.events.forEach(function (event) {
-      event.state_friendly = eventStates[event.state.status]
-      if (event.state_friendly) {
-        event.state_friendly = event.state_friendly.replace('AMOUNT', CURRENCY + (chargeData.amount / 100).toFixed(2))
-      }
       event.updated_friendly = dates.utcToDisplay(event.updated)
+      event.amount_friendly = event.amount ? asGBP(event.amount) : ''
+
+      const state = `${_.get(event, 'state.status')}`.toLowerCase()
+
+      if (!event.type || event.type.toLowerCase() === 'payment') {
```

Comment:
I am not sure I understand the rational here - maybe if you could explain it tomorrow as I am sure I am missing something.